project/Protobuf.scala

```diff
@@ -33,7 +33,7 @@ object Protobuf {
     // this keeps intellij happy for files that use the shaded protobuf
     Compile / unmanagedJars += (LocalProject("akka-protobuf-v3") / assembly).value,
     protoc := "protoc",
-    protocVersion := "3.11.4",
+    protocVersion := "3.21.12", // also sync with protobufJavaVersion in Dependencies.scala
```

Comment:
protoc download of this version, unless you install it in some other way: https://github.com/protocolbuffers/protobuf/releases/tag/v21.12